src/content/self-hosted/manage/crds.mdx

```diff
@@ -68,3 +68,26 @@ spec:
       key: <field in referenced secret> # key of the secret which contains the access secret key
       name: <secret-name> # name of the secret where the access secret is stored
 ```
+
+## Catalog Items
+
+Catalog items are mainly managed via the [Admin dashboard](admin/dashboard.mdx#catalog-items), but can also be managed via one of the CRDs installed by Okteto: `catalogitems.git.okteto.com`. This last case is useful, for example, if you want to add catalog items manually with a simple `kubectl`, and without the use of the Okteto UI.
```

Comment:
More than `kubectl`, I would mention the scenario to manage your catalog using GitOps